model_compression_toolkit/core/common/similarity_analyzer.py

```diff
@@ -41,74 +41,75 @@ def tensor_norm(x: np.ndarray, p: float = 2.0) -> np.float:
     """
     Compute the Lp-norm of a tensor x.
     Args:
-        x: Tensor to compute its norm
+        x: Flattened tensor to compute its norm
```

Comment:
Norm is useful for all kind of commutation. I think the flatten should be here and not force user to do it outside.